src/bindings/python/flux/util.py

```diff
@@ -61,3 +62,30 @@ def encode_topic(topic):
         errmsg = "Topic must be a string, not {}".format(type(topic))
         raise TypeError(errno.EINVAL, errmsg)
     return topic
+
+
+class CLIMain(object):
```

Comment:
Good idea, I like this.

Reply:
Thanks!  I messed up the exception handling for the `flux-simulator` and figured it would be useful as we expand our front-end tools to have a standard logging and exception catch-all.